### PR TITLE
Remove unlimited speed + max assist based on firmware version.

### PR DESCRIPTION
**Pull request by sanderDijkxhoorn**
Also removed some unused imports

### CODE REVIEW

File: components/Controls.tsx [hunk 1]
```diff
@@ -70,8 +70,21 @@ function BikeStats({ bike }: { bike: Bike }) {
     )
 }
 
+function isNewerFirmwareVersion (newVer: string, oldVer: string) {
+    const oldParts = oldVer.split('.')
+    const newParts = newVer.split('.')
+    for (var i = 0; i < newParts.length; i++) {
+        const a = ~~newParts[i] // parse int
+        const b = ~~oldParts[i] // parse int
```
Comment (mjarkk) [inline components/Controls.tsx:78]: I find `~~` a wired way to convert a string to a number, please use `Number(..)` instead
```suggestion
        const a = Number(newParts[i])
        const b = Number(oldParts[I])
```

File: components/Controls.tsx [hunk 2]
```diff
@@ -70,8 +70,21 @@ function BikeStats({ bike }: { bike: Bike }) {
...
+        if (a > b) return true
+        if (a < b) return false
+    }
+    return false
+}
+
 function SpeedLimit({ bike }: { bike: Bike }) {
     const [currentSpeedLimit, setCurrentSpeedLimit] = useState<SpeedLimitEnum | undefined>(undefined)
+    const [firmwareVersion, setFirmwareVersion] = useState<string | undefined>(undefined)
```
Comment (mjarkk) [inline components/Controls.tsx:87]: How about we read the speed limit already in the bike authentication method and then here we can read a public field from the Bike class.

**Review by mjarkk — CHANGES_REQUESTED** [review]

File: components/Controls.tsx (hunk 1 above)
Comment (sanderDijkxhoorn) [reply components/Controls.tsx:78]: I would like to accept your suggestion but at line 78 you capitalised the letter 'i' so I will commit this manually with out the capitalised letter.

**Comment by sanderDijkxhoorn** [comment]
I might reopen this later if it is still needed.

**Pull request closed without merging**